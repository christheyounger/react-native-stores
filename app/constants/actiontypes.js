export const GET_STORES = "GET_STORES";
export const GOT_STORES = "GOT_STORES";
export const GET_STORES_FAIL = "GET_STORES_FAIL";
export const GET_STORE = "GET_STORE";
export const GOT_STORE = "GOT_STORE";
export const GET_STORE_FAIL = "GET_STORE_FAIL";
export const GET_STORE_BRANCHES = "GET_STORE_BRANCHES";
export const GOT_STORE_BRANCHES = "GOT_STORE_BRANCHES";
export const GET_STORE_BRANCHES_FAIL = "GET_STORE_BRANCHES_FAIL";
