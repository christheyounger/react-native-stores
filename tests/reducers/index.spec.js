import rootReducer from "../../app/reducers";
import { GET_STORE } from "../../app/constants/actiontypes";

describe('root reducer', () => {
  test('should reduce the reducers to one reducer', () => {
    const reducedStore = rootReducer({}, {type: GET_STORE});
    expect(reducedStore).toMatchSnapshot();
  })
})
