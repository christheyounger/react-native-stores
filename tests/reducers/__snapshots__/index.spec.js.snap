// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`root reducer should reduce the reducers to one reducer 1`] = `
Object {
  "stores": Object {
    "loading": true,
    "stores": Array [],
  },
}
`;
